src/marketing/prex-type-visualsim.md

```diff
@@ -16,7 +16,7 @@ The _Visual similarity_ recommendation type offers recommendations for other pro
 -  Material
 -  Style
 
-You do not need to do anything to the images in your catalog for this recommendation type to work. Adobe Sensei uses AI to process and analyze the imagery in your catalog and builds attributes used to determine visual similarities. Because this recommendation type is not applicable to most catalogs, it is not enabled by default. You must explicitly enable this recommendation type.
+Adobe Sensei uses AI to process and analyze the imagery in your catalog and builds attributes used to determine visual similarities. If you are deploying this recommendation type in a production environment, you do not need to do anything to the images in your catalog. If you are testing this recommendation type in a non-production environment, make sure your image URLs are publicly accessible. Because this recommendation type is not applicable to most catalogs, it is not enabled by default. You must explicitly enable this recommendation type.
```

Comment:
```suggestion
Adobe Sensei uses AI to process and analyze the imagery in your catalog and builds attributes used to determine visual similarities. If you are testing this recommendation type in a non-production environment, make sure your image URLs are publicly accessible. Because this recommendation type is not applicable to most catalogs, it is not enabled by default. You must explicitly enable this recommendation type.
```